app/src/main/java/com/infomaniak/mail/utils/UiUtils.kt

```diff
@@ -32,4 +35,22 @@ object UiUtils {
         from + percent * (to - from)
 
     fun formatUnreadCount(unread: Int) = if (unread >= 100) "99+" else unread.toString()
+
+    fun fillInUserNameAndEmail(nameTextView: TextView?, name: String?, emailTextView: TextView?, email: String) {
+        when {
+            email.isMe() -> {
```

Comment:
email.isMine i think i'll better or something similar

Reply:
Has currently been copied to other branches that needed it as well, therefore I don't think it's the best moment to change its name. Maybe this could go into its own PR when we don't need to use it as much as currently